packages/auth/src/grant/routes.test.ts

```diff
@@ -664,4 +668,273 @@ describe('Grant Routes', (): void => {
       scope.isDone()
     })
   })
+
+  describe('interaction', (): void => {
+    describe('interaction start', (): void => {
+      test('Interaction start fails if grant is invalid', async (): Promise<void> => {
+        const scope = nock(KEY_REGISTRY_ORIGIN)
+          .get(TEST_KID_PATH)
+          .reply(200, {
+            ...TEST_CLIENT_DISPLAY,
+            keys: [
+              {
+                ...TEST_CLIENT_KEY.jwk,
+                revoked: false
+              }
+            ]
+          })
+
+        const ctx = createContext(
+          {
+            headers: {
+              Accept: 'application/json',
+              'Content-Type': 'application/json'
+            }
+          },
+          { interactId: 'unknown_interaction' }
+        )
+
+        await expect(grantRoutes.interaction.get(ctx)).resolves.toBeUndefined()
+        expect(ctx.status).toBe(401)
+        expect(ctx.body).toEqual({ error: 'unknown_request' })
```

Comment:
Does the spec say anything about the return codes here?

Reply:
It doesn't seem to specify the return code, just the body: https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol#section-3.6